contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,134 @@
+
+### Billing
+
+Billing is a major focus recently because we believe we can provide a better offering to users if we [charge for separate products](/handbook/engineering/feature-pricing). This leads to growth across the signup, usage, and revenue metrics the growth team cares about. Implementing it is on the growth team’s plate, and it’s a big task.
+
+It includes creating a [billing service](https://github.com/PostHog/posthog/pull/11986) enabling different pricing for different plans and [products](https://github.com/PostHog/posthog/pull/15035/files) such as session replays. It also means improving the billing page both on the [website](https://github.com/PostHog/posthog.com/pull/5181) and [in-app](https://github.com/PostHog/posthog/pull/14652) to work with this service.
+
+### Signup and activation
+
+Our growth team are improving the signup funnel by running A/B tests of options like [single sign on buttons](https://github.com/PostHog/posthog/pull/13234), [verifying emails](https://github.com/PostHog/posthog/pull/13632), and adding two-factor authentication.
+
+![Experiment example](../images/blog/what-is-a-growth-engineer/experiment.png)
+<Caption>An example of the pricing call to action experiment.</Caption>
+
+The results of this include:
+
+- Asking for billing details earlier led to a 250% increase in card activations.
+- A pricing call to action change led to a 27% increase in clickthroughs.
+- Focusing on the free plan on the pricing page led to a 14% increase in signups from that page.
+
+### Onboarding
+
+Onboarding is an area that is traditionally a big focus for growth teams, but isn't for us at the moment (it is a future priority).
+
+Our team has set up [in-app messaging](https://github.com/PostHog/posthog/pull/13108) and notifications, using customer.io.They have also added [autocapture](https://github.com/PostHog/posthog/pull/14687) and session recording toggles to the onboarding flow, increasing the visibility of these features.
+
+## Why you should hire a growth engineer
+
+Growth engineers help your overall business by helping all products grow faster. Growth metrics like signups and activation have downstream consequences on metrics other teams care about like feature usage and revenue.
+
+We've seen this from our growth team. Building cross-product features, like billing, has improved our overall business and user experience. These features don't fit into a specific product team's responsibility and would be missed out on if growth engineers didn't exist.
+
+Growth engineers likely won't help you find traction, they need products to connect and optimize to thrive. Once you have traction, it might be the right time to hire a growth engineer to fuel rapid growth.
```

Comment:
> Growth engineers likely won't help you find traction

This is certainly true, and in fact, growth engineers wont help you find product market fit AND you should only hire them once you have pmf